common-money.lic

```diff
@@ -29,7 +29,11 @@ module DRCM
   end
 
   def wealth(hometown)
-    currency = get_data('town')[hometown]['currency']
+    if hometown =~ /kronars|lirums|dokoras/
```

Comment:
Technically, see note below. I went through and made sure all the cases were the same.